### ISSUE TITLE
VANotify New Copay Statement Email Content Review from Debt Resolution Team (MCP)

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#32315]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [ ] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [ ] Making revisions to an existing online form, app, or tool on VA.gov
- [x] Something else (please add details in the next question)

## What do you need help with?
We wrote a draft of the email that a Veteran will receive if they have a new copay statement available at a facility. We used an example email template from the VANotify team, but it didn't 100% align with the example so I wanted to get this reviewed. 
  
## Supporting Artifacts

- Link to product outline:
- Link to or attach user flows:
- Link to prototype or mockups: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/Medical_Copays/notifications/new-copay-statement.md
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

## When does this work need to be done?
- Estimated launch date: TBD
- Estimated staging review date:
- Content and IA work needed by: 

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner: Matt Self
-	Product manager: Denise Coveyduc
-	Designer: Rebecca Walsh, Riley Orr
-	FE engineer: Craig Wheeler, Justin Lin
-	Product/team Slack channel: vsa-debt

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) Slack channel and tag @RLHecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by denisecoveyduc** [comment]
@DanielleThierryUSDSVA @RLHecht Have you had a chance to review this email draft? I'd like to move forward with the VA Notify team.

**Comment by RLHecht** [comment]
@denisecoveyduc We have 3 edits:

1. Let's change the first sentence to active voice: We updated your health care copay balance for ((facility name)) on ((patient statement date)).
2. Let's remove the "Thank you for your service" sign-off. It's odd to have a website thanking someone for their service.
3. Let's use contractions in the sentence underneath the sign-off (we do this as part of our plain language standards): You're receiving this email because you submitted a request on VA.gov. Please don't reply to this email. If you need to contact us, please visit VA.gov.

Everything else looks great!

**Comment by rileyorr** [comment]
Thanks for the feedback @RLHecht! I've incorporated your changes. 

Additionally, I removed the facility name from the email based on feedback from the VA Notify team that we shouldn't put too much identifying information in our notifications. The sentence now reads: We updated your health care copay balance on ((patient statement date)).

It everything looks good I'll go ahead and send this over to our stakeholders to review it with their privacy officer!
